tests/rules/no-unused-expressions.js

```diff
@@ -136,8 +136,9 @@ ruleTester.run("no-unused-expressions", rule, {
         },
 
         // Babel-specific test cases.
-      { code: "let a = do { foo; let b = 2; }", errors: [{ message: "Expected an assignment or function call and instead saw an expression.", type: "ExpressionStatement" }] },
+        { code: "let a = do { foo; let b = 2; }", errors: [{ message: "Expected an assignment or function call and instead saw an expression.", type: "ExpressionStatement" }] },
         { code: "let a = do { if (foo) { foo.bar } else { a; bar.foo } }", errors: [{ message: "Expected an assignment or function call and instead saw an expression.", type: "ExpressionStatement" }] },
+        { code: "function foo() { foo.bar?.(); }", errors: [] },
```

Comment:
Shouldn't this be `foo.bar?.baz()`? What does this mean without a method name identifier?

Reply:
Ah, also, this is valid, correct? If so, it should be in the `valid` section of this test suite.

Reply:
This means, `foo.bar` is a normal GET, but `if `foo.bar` is nullish, it won't be invoked.

Reply:
Cool, thanks for the explanation! Do you feel like this should be the default test case for this?

Reply:
i don't know why it needs to be in a function; but anything that verifies that `foo.bar?.()` is the same re unused expressions as `foo.bar()` seems fine

Reply:
You're both right, thanks! Just to clarify: This test is specifically about optional calls, not optional chaining in general.